features/background-sync.yml

```
@@ -0,0 +1,13 @@
+name: Background sync
+description: The background synchronization API registers a service worker callback to run only when the device is no longer offline. You can use this to let users continue using your app while offline, and synchronize with a server later when, when a connection is established.
```

Fixing the double "when" nudged me toward this suggestion for brevity. In any case, if the double "when" is fixed, then this is ready to merge.

```suggestion
description: The background synchronization API registers a service worker callback to run only when the device is no longer offline. You can use this to let users continue using your app while offline and synchronize with a server after reconnecting.
```